packages/engine/lib/memory/src/shared_memory/shared_memory.rs

```diff
@@ -13,7 +13,10 @@ use crate::{
     },
 };
 
-/// An identifier for a shared memory section.
+/// An identifier for a shared memory [`Segment`].
+///
+/// Holds a UUID and a random suffix. The Uuid can be reused for different [`Segment`]s and can all
```

Comment:
```suggestion
/// Holds a UUID and a random suffix. The UUID can be reused for different [`Segment`]s and can all
```

Reply:
(Or go the other way if you're trying to be consistent with the type)

Reply:
`Uuid` in code and "UUID" in docs :sli